sources/models/shared-defaults/defaults.toml

```diff
@@ -119,6 +119,7 @@ template-path = "/usr/share/templates/hosts"
 
 [settings.ntp]
 time-servers = ["169.254.169.123", "2.amazon.pool.ntp.org"]
+options = ["iburst"]
```

Comment:
@yeazelm, this looks correct to me. The options setting is added by the migration and the defaults file will now include "iburst" such that behavior does not change on upgrade.

Reply:
This looks right to me!